docs: split telemetry and troubleshooting sections

charislam opened the pull request:
Split telemetry and troubleshooting sections, and promote troubleshooting up to global nav.

charislam committed:
docs: split telemetry and troubleshooting sections

gregnr commented on apps/docs/content/guides/telemetry.mdx:
```diff
@@ -0,0 +1,5 @@
+title: Telemetry
+---
+
+Telemetry is the process of collecting data about your app. Good telemetry lets you identify and fix issues quickly by informing you of errors and their contexts. This section provides guidance on monitoring errors in Supabase, including how to work with Supabase logs.
```
Worth linking to the logging & observability subsections from this page?

kiwicopple replied:
also some suggested copy:

```suggestion
Telemetry helps you understand what’s happening inside your app by collecting logs, metrics, and traces.

* **Logs** capture individual events, such as errors or warnings, providing details about what happened at a specific moment.
* **Metrics** track numerical data over time, like request latency or database query performance, helping you spot trends.
* **Traces** show the flow of a request through different services, helping you debug slow or failing operations.

Supabase is working towards full support for the [OpenTelemetry](https://opentelemetry.io/) standard, making it easier to integrate with observability tools. 

This section provides guidance on monitoring in Supabase, including how to work with Supabase Logs.
```

gregnr reviewed (approved):
Nice 👍

charislam committed:
docs: update telemetry copy

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

charislam committed:
fix(ui): fix troubleshooting view in mobile

charislam committed:
format

charislam committed:
fix: troubleshooting search

Troubleshooting previews are displayed using a grid now, and hidden
doesn't work (overriden by grid behavior). Instead, applying
display:none when filtered out.

charislam committed:
sync to db
